src/vpn-worker.ts

```diff
@@ -117,11 +114,11 @@ const worker = (instanceId: number, serviceId: number) => {
 	});
 
 	vpn.on('process:exit', (code, signal) => {
-		let msg = 'process exited';
+		let msg = `openvpn process (pid=${vpn.pid()})`;
```

Comment:
This will now be the message in the case that no code/signal is emitted.  I've got no idea if that's ever actually the case but if so it's probably worth keeping the verb of what happened

Reply:
Nice catch, fixed.